ztp/gitops-subscriptions/argocd/example/policygentemplates/group-du-sno-ranGen.yaml

```diff
@@ -184,3 +184,18 @@ spec:
 #                user:
 #                  name: root
 #     ---- sources needed for updating CRI-O workload-partitioning ends here ----
+
+    # ---- Sources needed to configure IPsec encryption for external traffic ----
+    # 1. Provide the following certificate files:
+    #    * left_server.p12
+    #    * ca.pem
+    # 2. Set the following values in the `ipsec-endpoint-config.yml`:
+    #    * `$clusterNode` - the node IP address or DNS hostname
+    #    * `$externalHost` - the external host IP address or DNS hostname
+    #    * `$externalAddress` - the IP address or subnet of the external host
+    # 3. Run the `build.sh` script to create the machine config
+    # - fileName: enable-ipsec.yaml
```

Comment:
This file path does not align with what we added to the source-crs, so the deployment will fail due to file does not exist. 

I think this the file path ztp can recognize, however, usually "extra-manfest" gets applied at install time, so there are some inconsistencies here as we are trying too apply these at day2 config time.

`optional-extra-manifest/ipsec/ipsec-endpoint-config.yml`